nhrpd/nhrp_shortcut.c

```diff
@@ -51,18 +51,26 @@ static int nhrp_shortcut_do_expire(struct thread *t)
 static void nhrp_shortcut_cache_notify(struct notifier_block *n,
 				       unsigned long cmd)
 {
+	char buf2[PREFIX_STRLEN];
+
 	struct nhrp_shortcut *s =
 		container_of(n, struct nhrp_shortcut, cache_notifier);
+	struct nhrp_cache *c;
 
+	c = s->cache;
```

Comment:
nitpick, can you move the assignment where  you declare c ?

Reply:
done.